lib/old-api.js

```diff
@@ -397,7 +397,12 @@ function processHTML(config) {
   } else if (window.document.readyState === "complete") {
     scriptComplete();
   } else {
-    window.addEventListener("load", scriptComplete);
+    function onWindowLoad() {
+      window.removeEventListener("load", onWindowLoad);
+      scriptComplete();
+    }
+
+    window.addEventListener("load", onWindowLoad);
```

Comment:
You could do `window.addEventListener("load", scriptComplete, { once: true })`, I think.